data_spec_validator/decorator/decorators.py

```diff
@@ -1,19 +1,50 @@
 from functools import wraps
 from typing import Dict, List, Union
 
+from data_spec_validator.spec import DSVError, raise_if, validate_data_spec
+
 try:
-    import rest_framework.exceptions as drf_exceptions
     from django.core.handlers.wsgi import WSGIRequest
-    from django.http import QueryDict
+    from django.http import HttpResponseBadRequest, HttpResponseForbidden, QueryDict
     from django.views.generic.base import View
+except ModuleNotFoundError as e:
+    print(f'[DSV][WARNING] decorator: "dsv" cannot be used, {e}')
+
+_enabled_drf = False
+try:
+    import rest_framework.exceptions as drf_exceptions
     from rest_framework.request import Request
+
+    _enabled_drf = True
 except ModuleNotFoundError as e:
     print(f'[DSV][WARNING] decorator: "dsv" cannot be used, {e}')
```

Comment:
nit:
consider revise to `[DSV][INFO] decorator: using "dsv" without drf`

Reply:
revised, thanks!